import unittest

>>> add_integer = __import__('0-add_integer').add_integer
>>> print(add_integer(1, 2))
3
>>> print(add_integer(100, -2))
98
>>> print(add_integer(2))
100
>>> print(add_integer(100.3, -2))
98
>>> print(add_integer(4, "School"))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 35, in add_integer
    raise TypeError("b must be an integer")
TypeError: b must be an integer
>>> print(add_integer(None))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 33, in add_integer
    raise TypeError("a must be an integer")
TypeError: a must be an integer
>>> print(add_integer(2, None))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 35, in add_integer
    raise TypeError("b must be an integer")
TypeError: b must be an integer
>>> print(add_integer(None, 2))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/root/alx-higher_level_programming/0x07-python-test_driven_development/0-add_integer.py", line 33, in add_integer
    raise TypeError("a must be an integer")
TypeError: a must be an integer
>>> print(add_integer(None, 2, 5))
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: add_integer() takes from 1 to 2 positional arguments but 3 were given
